test/Extensions/ServiceBus.Tests/EvictionStrategyTests/EHPurgeLogicTests.cs

```diff
@@ -226,22 +227,15 @@ private static async Task AddDataIntoCache(EventHubQueueCacheForTesting cache, i
 
         private static EventData MakeEventData(long sequenceNumber)
         {
-            byte[] ignore = { 12, 23 };
             var now = DateTime.UtcNow;
-            var eventData = new TestEventData(ignore,
-                offset: now.Ticks,
+            var eventData = EventHubsModelFactory.EventData(
+                eventBody: new BinaryData([12, 23]),
```

Comment:
We could do `EventData.Empty` here, but I kept the existing payload to avoid unintentional changes.